pom.xml

```diff
@@ -109,6 +109,7 @@
                 <artifactId>maven-checkstyle-plugin</artifactId>
                 <version>${checkstyle.plugin.version}</version>
                 <configuration>
+                    <includeTestSourceDirectory>true</includeTestSourceDirectory>
                     <configLocation>code-style/checkstyle.xml</configLocation>
```

Comment:
I think we should include test and src resources as well.

Reply:
Have done this manually for the time being.  Amounts to checking the files each have the licence header.